l10n_it_ricevute_bancarie/report/distinta_qweb.py

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8 -*-
+##############################################################################
```

Comment:
Change header with this model: https://github.com/OCA/maintainer-tools/blob/master/template/module/__init__.py#L1